sunpy/visualization/tests/test_drawing.py

```diff
@@ -54,6 +69,14 @@ def test_heliographic_equator_prime_meridian(heliographic_test_map):
     drawing.prime_meridian(axes, color="red")
 
 
+@figure_test
+def test_draw_extent(cropped_aia193_sample_map, euvi195_sample_map):
+    fig = plt.figure()
+    ax = fig.add_subplot(projection=euvi195_sample_map)
+    euvi195_sample_map.plot(axes=ax)
+    drawing.extent(ax, cropped_aia193_sample_map.wcs)
```

Comment:
```suggestion
    drawing.extent(ax, cropped_aia193_sample_map.wcs)
    return fig
```

Reply:
Why? None of the other tests return `fig`

Reply:
My preference is to return `fig`, and to *also* instantiate `fig` without using pyplot: `fig = Figure()`